crates/mipsy_lib/src/compile/mod.rs

```diff
@@ -60,6 +60,21 @@ impl Breakpoint {
     }
 }
 
+pub trait Point {
+    fn get_id(&self) -> u32;
+    fn get_commands(&'_ mut self) -> &'_ mut Vec<String>;
+}
+
+impl Point for Breakpoint {
```

Comment:
Why is the `impl Point for Breakpoint` in `crates/mipsy_lib/src/compile/mod.rs` and `impl Point for Watchpoint` in `crates/mipsy_interactive/src/interactive/commands/mod.rs`?
They seem like they should be in similar locations.

Reply:
mipsy_web may want to implement watchpoints.
So having `impl Point for Watchpoint` in lib also might be a good idea.

Reply:
ah I hadn't considered this, I had `Watchpoint` in `mipsy_interactive` as I didn't think `mipsy_lib` would have a use for them, but this is definitely worth adding to `mipsy_web`